### PR TITLE
ign -> gz Provisional Finale: Source Migration : gz-utils

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/767

### CODE REVIEW

File: cli/include/gz/utils/cli/GzFormatter.hpp [hunk 1]
```diff
@@ -167,4 +168,7 @@ private: std::unordered_multimap<std::string, std::string> needs;
 private: std::unordered_multimap<std::string, std::string> needed_by;
 };
 
+// TODO(CH3): Deprecated. Remove on tock.
+using usingIgnitionFormatter GZ_DEPRECATED(2) = GzFormatter;
```
Comment (chapulina) [inline cli/include/gz/utils/cli/GzFormatter.hpp:172]:
```suggestion
using IgnitionFormatter GZ_DEPRECATED(2) = GzFormatter;
```

**Review by chapulina — CHANGES_REQUESTED** [review]

File: cli/include/gz/utils/cli/GzFormatter.hpp (hunk 1 above)
Comment (methylDragon) [reply cli/include/gz/utils/cli/GzFormatter.hpp]: Oops, nice catch!

https://github.com/gazebosim/gz-utils/compare/9e113499fd3953025ef626842437ec91c7473964..818aed629ac4753e7cf0a5d84ff0235eb7647c9c

I suspect the `GZ_DEPRECATED` call will cause Windows to throw a fit, let me know if it's ok to remove if it does.

Comment (chapulina) [reply cli/include/gz/utils/cli/GzFormatter.hpp]: Let's add ifdefs so the deprecation isn't defined on Windows for these cases. Windows isn't an officially supported platform, so we can do a hard-tock here and there.

Comment (methylDragon) [reply cli/include/gz/utils/cli/GzFormatter.hpp]: https://github.com/gazebosim/gz-utils/pull/70/commits/d179ba3b6563e3c1c11a3a3e97a169a697ad82cb !

**Review by chapulina — APPROVED** [review]

**Pull request merged**